### DISCUSSION TITLE
Server build not minimized

### DISCUSSION THREAD

**Discussion by mattrunyon** [facebook/docusaurus#9231] [Q&A] [answered]
Just curious, is there a reason the server builds are not minimized? I see that `minimizeEnabled = minify && isProd && !isServer` in the webpack base config

I noticed this while writing a plugin to switch the minifier to esbuild

I see a comment saying "Only minimize client bundle in production because server bundle is only used for static site generation". Not sure why that would be the case

**Comment by slorber** [comment, marked as answer]
The server bundle is only used during SSR, and at the end we delete it. It is only a temporary file that is never sent to the browser, and IMHO there's no reason to minimize it. Why should we minimize it?

**Reply by mattrunyon** [reply]
That makes sense. I misunderstood and thought the server bundle was part of what the browser gets on first load before the page fully hydrates
